mesonbuild/backend/vs2010backend.py

```diff
@@ -151,7 +151,11 @@ def generate_custom_generator_commands(self, target, parent_node):
                     args = [x.replace("@INPUT@", infilename).replace('@OUTPUT@', sole_output)
                             for x in base_args]
                     args = self.replace_outputs(args, target_private_dir, outfiles_rel)
-                    args = [x.replace("@SOURCE_DIR@", self.environment.get_source_dir()).replace("@BUILD_DIR@", target_private_dir)
+                    args = [x.replace("@SOURCE_DIR@", self.environment.get_source_dir())\
```

Comment:
__[flake8]__
* [E502] the backslash is redundant between brackets

<!-- Comment by [SideCI](https://sideci.com) -->